src/aura/CRLP_FilterGroup/CRLP_FilterGroupController.js

```diff
@@ -91,7 +91,10 @@
     /**
      * @description: cancels the pop up for filter rule and clears the active filter rule
```

Comment:
small add here: need to update the comment with new functionality. You also won't need to run resetActiveFilterRule or set the filterRuleMode if mode == delete since those only apply to the filter rule modal settings.

Reply:
also found a bug likely from here: hit the cancel button when adding or editing a filter rule and edit mode is being removed.

Reply:
Found that one as well